library/std/src/sys/pal/hermit/os.rs

```diff
@@ -14,15 +14,15 @@ use crate::sys::unsupported;
 use crate::vec;
 
 pub fn errno() -> i32 {
-    0
+    unsafe { abi::get_errno() }
 }
 
-pub fn error_string(_errno: i32) -> String {
-    "operation successful".to_string()
+pub fn error_string(errno: i32) -> String {
+    abi::error_string(errno).to_string()
```

Comment:
Is it not this?
```suggestion
    abi::errno::error_string(errno).to_string()
```

Reply:
Both. I will be also exported by [ src/lib.rs](https://github.com/hermit-os/hermit-rs/blob/main/hermit-abi/src/lib.rs#L13). Maybe a bad solution. `errno` was declared as public module and I don't want to change it. Maybe it will be used by other projects.

Reply:
I'm happy either way.